README.md

```diff
@@ -1 +1,21 @@
 Code for persisting ROS message data using MongoDB.  Contains C++ and Python libraries to serialize ROS data to MongoDB, as well as some handy scripts to record data from the command line.  Based on code split out of warehouse_ros.
+
+# Building from source
+
+## ROS Jade 
+
+In order to build from source you'll need to install the [mongo c++ drivers](https://github.com/mongodb/mongo-cxx-driver/wiki/Download-and-Compile-the-Legacy-Driver)
+
+First get the driver:
+```
+git clone -b 26compat https://github.com/mongodb/mongo-cxx-driver.git
+```
+
+Then compile using scons:
+```
+sudo apt-get install scons
+scons --prefix=/usr/local/ --full --use-system-boost --disable-warnings-as-errors
```

Comment:
this should also be sudo (unless your on Docker) as already corrected http://moveit.ros.org/install/source/